### PR TITLE
✨ Add Case Studies to CMS

### PR DESCRIPTION
**Pull request by peduarte**
Adding Case Studies to CMS.

Plus a few minor tidy ups.

### CODE REVIEW

**Comment by peduarte** [comment]
Deploy preview for *fathomlondon* ready!

Built with commit 2ca157c2f7773886935b3e6ccadd83b1361eebe9

https://deploy-preview-255--fathomlondon.netlify.com

File: src/cms/preview-templates/CaseStudyPostPreview.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { PostBody } from '../../components/PostBody';
+
+const BlogPostPreview = ({ entry, widgetFor }) => (
```
Comment (artisologic) [inline src/cms/preview-templates/CaseStudyPostPreview.js]: Should that be called `CaseStudyPostPreview`?

Comment (peduarte) [reply src/cms/preview-templates/CaseStudyPostPreview.js]: Yes!

**Comment by peduarte** [comment]
@artisologic Updated

File: static/admin/config.yml [hunk 2]
```diff
@@ -22,6 +22,20 @@ collections:
       - { label: "Hero", name: "hero", widget: "image" }
       - { label: "Author", name: "author", widget: "string" }
       - { label: "Body", name: "body", widget: "markdown" }
+  - name: "case_studies"
+    label: "Case studies"
+    label_singular: "Case study"
+    folder: "content/case-studies"
```
Comment (AndrewJDick) [inline static/admin/config.yml]: Is there any reason why we are calling it `case_studies` in some instances, and `case-studies` in others?

Comment (peduarte) [reply static/admin/config.yml]: @AndrewJDick yes, the Netlify CMS convention seems to use `_` (as per the docs https://www.netlifycms.org/docs/configuration-options/)

So to be consistent, I'm using `_` too

**Pull request merged**